### PR TITLE
Fix build failure caused by incorrect NODE_PATH value

### PR DESCRIPTION
**Pull request by tkhanna42**
# Description

TL:DR; Setting `NODE_PATH=./src` in `package.json` seems to work, and fixes the build error locally. Using `cross-env` makes it compatible with non Linux/Mac machines

So apparently [some linux machines have `NODE_PATH` set to a value as a global environment variable, and any other local .env variables don't overwrite them.](https://github.com/facebook/create-react-app/issues/3939#issuecomment-383331530)

I verified this on my machine by running `printenv | grep NODE_PATH` and got the output:
```
NODE_PATH=/usr/local/lib/node_modules
```

Also, using `NODE_PATH=src` from my `.env` file suddenly stopped working with `npm start`, and I couldn't figure out why... There also seems to be related to changes in create-react-app (see issues [3939](https://github.com/facebook/create-react-app/issues/3939) and currently open [6021](https://github.com/facebook/create-react-app/issues/6021), didn't see anything in the changelog since 1.1.0 saying it was fixed) but not sure if its still relevant.

I noticed people using `cross-env` didn't have this issue, and so tried just setting the `NODE_PATH=.src/` in `package.json` and it looks like it works ok now. Hopefully it also passes on travis.

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

Tested with `npm start` and `npm run build` locally

# Checklist:

- [x] I have performed a self-review of my own code

### CODE REVIEW

**Comment by theRoughCode** [comment]
Sorry for the late response.  Thanks for the fix!

**Comment by tkhanna42** [comment]
Np! (I kinda introduced the issue, haha)Thanks for approving!
